src/CoreWCF.Primitives/src/CoreWCF/Description/ServiceDescription.cs

```diff
@@ -231,6 +243,26 @@ private static void GetIServiceBehaviorAttributes(Type currentServiceType, Keyed
             }
         }
 
+        internal static void SetupSingleton<TService>(ServiceDescription serviceDescription, IServiceProvider services) where TService : class
+        {
+            ServiceBehaviorAttribute serviceBehavior = serviceDescription.Behaviors.Find<ServiceBehaviorAttribute>();
+            Debug.Assert(serviceBehavior != null, "EnsureServiceBehavior should have ensured the serviceBehavior");
```

Comment:
should we these message to resource.xml or some format... or you want to keep as it it